### PR TITLE
UI: Bugs + Test Coverage

### PR DESCRIPTION
**Pull request by shangyian**
### Summary

The changes include:
* Display all column attributes for each column in the node info page's column tab
* Fix a few bugs including:
  * PK not passed in correctly during editing
  * PK not displaying properly on node page
  * Node page displaying "Node not found" prematurely (i.e., before the fetch call has had a chance to respond)
* Fix warnings with tests and errors where the dom on different tests was interacting with each other and causing failures. 

Test coverage is significantly improved from before:
```
BEFORE:
Jest: "global" coverage threshold for statements (90%) not met: 20.37%
Jest: "global" coverage threshold for branches (90%) not met: 21.15%
Jest: "global" coverage threshold for lines (90%) not met: 20.96%
Jest: "global" coverage threshold for functions (90%) not met: 15.14%
```
```
AFTER:
Jest: "global" coverage threshold for statements (100%) not met: 90.23%
Jest: "global" coverage threshold for branches (100%) not met: 78.93%
Jest: "global" coverage threshold for lines (100%) not met: 90.92%
Jest: "global" coverage threshold for functions (100%) not met: 87.8%
```

### Test Plan

- [ ] PR has an associated issue: #
- [ ] `make check` passes
- [ ] `make test` shows 100% unit test coverage

### CODE REVIEW

File: datajunction-ui/src/app/pages/NodePage/__tests__/NodeLineageTab.test.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    await waitFor(() => {
+      expect(djClient.DataJunctionAPI.node_lineage).toHaveBeenCalledWith(
+        defaultProps.djNode.name,
+      );
+
+      // The origin node should be displayed
+      expect(screen.getByText('Default: Avg Repair Price')).toBeInTheDocument();
+
+      // Every node in the DAG should be displayed on the screen
```
Comment (agorajek) [inline datajunction-ui/src/app/pages/NodePage/__tests__/NodeLineageTab.test.jsx:52]: What's the reason for commenting this out?

File: datajunction-ui/src/app/pages/SQLBuilderPage/__tests__/index.test.jsx [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    fireEvent.keyDown(selectDimensions.firstChild, { key: 'ArrowDown' });
+
+    for (const dim of mockCommonDimensions) {
+      expect(screen.getAllByText(dim.name)[0]).toBeInTheDocument();
+      fireEvent.click(screen.getAllByText(dim.name)[0]);
+    }
+    expect(mockDjClient.sqls).toHaveBeenCalled();
+  });
+
+  // it('handles Run Query button click', () => {
```
Comment (agorajek) [inline datajunction-ui/src/app/pages/SQLBuilderPage/__tests__/index.test.jsx:174]: ditto. Let's delete this if ti shouldn't be here.

**Review by agorajek — APPROVED** [review]
@shangyian I am sorry for the late review. Looks great. Just left two comments inline.

File: datajunction-ui/src/app/pages/NodePage/__tests__/NodeLineageTab.test.jsx (hunk 1 above)
Comment (shangyian) [reply datajunction-ui/src/app/pages/NodePage/__tests__/NodeLineageTab.test.jsx]: good catch, removed!

**Pull request merged**